### PR TITLE
OKTA-433579 Polling Push Factor Verification Using .NET SDK

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**
## Issue \#

#507 

## Code

- [x] Unit test(s)
- [x] Implementation

## Desired behavior

Add a property with the TransactionId from "_links" in Push Factor Verification response
Example of the response https://developer.okta.com/docs/reference/api/factors/#issue-a-push-factor-challenge

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Running changelog of releases since `3.1.1`
 
 
+## v5.3.1
+
+### Features
+
+- Add `GetTransactionId` function in `VerifyUserFactorResponse`. The function makes sense for Push Factor verification responses which contain this Id. The return value can further be used with `GetFactorTransactionStatusAsync` operation. See [Issue a Push Factor challenge](https://developer.okta.com/docs/reference/api/factors/#issue-a-push-factor-challenge) for details. (#507)
```
Comment (bryanapellanes-okta) [inline CHANGELOG.md:9]:
```suggestion
- Add `VerifyUserFactorResponse.GetTransactionId` method to simplify access to the transaction ID of the `VerifyUserFactorResponse` where appropriate. The transaction ID is further used with the `GetFactorTransactionStatusAsync` operation. See [Issue a Push Factor challenge](https://developer.okta.com/docs/reference/api/factors/#issue-a-push-factor-challenge) for details. (#507)
```

**Review by bryanapellanes-okta — COMMENTED** [review]
Minor suggestion; should probably be reviewed by docs team member.

**Review by bryanapellanes-okta — APPROVED** [review]
👍

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

**Pull request merged**